test/unit/lib/plugins/aws/invoke.test.js

```diff
@@ -454,7 +456,143 @@ describe.skip('test/unit/lib/plugins/aws/invoke.test.js', () => {
+      InvocationType: 'RequestResponse',
+      LogType: 'None',
+      Payload: Buffer.from('{}'),
+    });
+  });
+
+  it('should support `--context` param with `--raw` param', async () => {
+    const lambdaInvokeStub = sinon.stub();
+
+    const result = await runServerless({
+      fixture: 'invocation',
+      cliArgs: ['invoke', '--function', 'callback', '--raw', '--context', '{"ctx": "somecontext"}'],
+      awsRequestStubMap: {
+        Lambda: {
+          invoke: args => {
+            lambdaInvokeStub.returns('payload');
+            return lambdaInvokeStub(args);
+          },
+        },
+      },
+    });
+    expect(lambdaInvokeStub).to.have.been.calledOnce;
+    expect(lambdaInvokeStub.args[0][0]).to.deep.equal({
+      ClientContext: 'IntcImN0eFwiOiBcInNvbWVjb250ZXh0XCJ9Ig==', // "{\"ctx\": \"somecontext\"}"
+      FunctionName: result.serverless.service.getFunction('callback').name,
+      InvocationType: 'RequestResponse',
+      LogType: 'None',
+      Payload: Buffer.from('{}'),
+    });
+  });
+
+  it('should support `--contextPath` param', async () => {
+    const lambdaInvokeStub = sinon.stub();
+
+    const tmpDirPath = getTmpDirPath();
+    fse.mkdirSync(tmpDirPath);
+    const data = {
+      testProp: 'testValue',
+    };
+    const contextDataFile = path.join(tmpDirPath, 'context.json');
+    fse.writeFileSync(contextDataFile, JSON.stringify(data));
+
+    const result = await runServerless({
+      fixture: 'invocation',
+      cliArgs: ['invoke', '--function', 'callback', '--contextPath', contextDataFile],
+      awsRequestStubMap: {
+        Lambda: {
+          invoke: args => {
+            lambdaInvokeStub.returns('payload');
+            return lambdaInvokeStub(args);
+          },
+        },
+      },
+    });
+    expect(lambdaInvokeStub).to.have.been.calledOnce;
+    expect(lambdaInvokeStub.args[0][0]).to.deep.equal({
+      ClientContext: 'eyJ0ZXN0UHJvcCI6InRlc3RWYWx1ZSJ9', // {"testProp":"testValue"}
+      FunctionName: result.serverless.service.getFunction('callback').name,
+      InvocationType: 'RequestResponse',
+      LogType: 'None',
+      Payload: Buffer.from('{}'),
+    });
+  });
+
+  it('should throw error on invoke with contextPath if file not exists', async () => {
+    const lambdaInvokeStub = sinon.stub();
+
+    const contextDataFile = path.join(getTmpDirPath(), 'context.json');
+
+    await expect(
+      runServerless({
+        fixture: 'invocation',
+        cliArgs: ['invoke', '--function', 'callback', '--contextPath', contextDataFile],
+        awsRequestStubMap: {
+          Lambda: {
+            invoke: args => {
+              lambdaInvokeStub.returns('payload');
+              return lambdaInvokeStub(args);
+            },
+          },
+        },
+      })
+    ).to.be.rejectedWith('The file you provided does not exist.');
+    expect(lambdaInvokeStub).to.have.been.callCount(0);
+  });
+
+  it('should allow invoke with raw context for contextPath', async () => {
```

Comment:
By content this test looks as duplicate of `should support `--contextPath` param`.

What's the purpose of it?